subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -1975,7 +1975,7 @@ static int mod_sub_va(uint32_t op, uint16_t net_idx, uint16_t addr, uint16_t ele
 	}
 
 	BT_DBG("net_idx 0x%04x addr 0x%04x elem_addr 0x%04x label %s",
-	       net_idx, addr, elem_addr, log_strdup(label));
+	       net_idx, addr, elem_addr, log_strdup((const char *)label));
```

Comment:
Is `label` even even null-terminated? I don't think so - it's just a 128-bit "UUID", right @trond-snekvik ? So there's a bug in the original code that needs fixing and doing the type cast just hides the bug even more.

Reply:
Good point, yes, it doesn't have to be.

Reply:
👍

Reply:
I'm surprised that it's always there.